samples/cpp/image_generation/image2image_concurrency.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) 2023-2025 Intel Corporation
```

Comment:
Please add a couple of lines about these samples to readme

Reply:
Will do!

Reply:
Pushed the changes.